service.upnext/codecov.yml

```diff
@@ -0,0 +1,11 @@
+coverage:
```

Comment:
please remove from the submission. If you're using kodi-addon-submitter, add this to .gitattributes

Reply:
Thanks, added to .gitattributes